SofaKernel/modules/SofaHelper/src/sofa/helper/system/FileRepository.cpp

```diff
@@ -337,7 +346,7 @@ std::string FileRepository::relativeToPath(std::string path, std::string refPath
         /// Case sensitive OS.
         std::string::size_type loc = path.find( refPath, 0 );
         if (loc==0)
-            path = path.substr(refPath.size()+1);
+            path = path.substr(refPath.size());
```

Comment:
I'm not sure what returns ```path.find```
But if it is an index where the path has been found in the refPath. 
The following block shouldn't be:
``` 
if (loc == 0)
   path = refPath; // not relative found
else
   path = path.substr(loc);
```

Reply:
Hi @epernod 

Thank for the suggestion but I don't think it is what we want to do.
If loc == 0 means there is a match between path and refPath and this match  starts at the zero position.
when this happens we don't want to returns refPath but the string content following the common prefix (of size refPath.size()).